### ISSUE TITLE
Unable to edit polygon vertex

### ISSUE THREAD

**Issue by ananovosel** [doug-m/CollectorAuroraTesting#237]
### Issue
When editing polygon features, one can add new vertex but cannot edit the existing.

### Expected Behavior
One should be able to select each vertex and move it. It should work the same way as it does in Collector Classic.

### Device Information
* Device: Huawei P20
* OS Version: Android 9.1.0

**Comment by kyliedonia** [comment]
This should be possible in the app today. @ananovosel did you try tapping on a vertex to select it? Once selected, position the location target where you want the vertex moved to (either using the GPS or by panning the map) and then from the overflow menu choose "Update selected point" -- are you able to see/do this?

There is a bug with the overflow menu: it won't show up when you start editing, but if you make the panel smaller it then appears (and stays if you go to other panel sizes).

**Comment by dmcilwr** [comment]
This didn't work for me either, although I have used it in Collector Classic - so I was looking for it
I'm not getting an option to Update point for the selected vertex, no overflow menu and no ability to resize the panel

**Comment by kyliedonia** [comment]
To resize the panel, use the header (like where the symbol is) and drag it up/down. Is that not working for you @dmcilwr?

**Comment by trigonet** [comment]
Hello, for us it seems to be like this: Technically vertex editing is working, but it is not practicable. 

The editing of existing polygon points is therefore very difficult and not  user friendly. We often have the case that rectangular areas (for example buildings) are collected in the field with the help of GPS measurements. As the measurements are not exactly it is necessary to edit the shape afterwards, that it becomes more perfect rectangular. To perform this task you have to do the following:

Old sketch tool:
Select point
Move point to desired position

New sketch tool:
Select point
Move map to set centerpoint to the desired position (but confusing seeing a "new point" and corresponding lines)
Select menu
Select "Update selected point"
![Screenshot_20181105-132123](https://user-images.githubusercontent.com/53850565/62848000-b01b1200-bcd9-11e9-9574-e7e75e41b660.jpg)
![Screenshot_20181105-132325](https://user-images.githubusercontent.com/53850565/62848004-b3ae9900-bcd9-11e9-9b93-f725b0d2fe94.jpg)

**Comment by ananovosel** [comment]
> This should be possible in the app today. @ananovosel did you try tapping on a vertex to select it? Once selected, position the location target where you want the vertex moved to (either using the GPS or by panning the map) and then from the overflow menu choose "Update selected point" -- are you able to see/do this?
> 
> There is a bug with the overflow menu: it won't show up when you start editing, but if you make the panel smaller it then appears (and stays if you go to other panel sizes).

Yes, now I managed to edit polygon by moving and deleting existing points, but it is really not intuitive or practical. And also, if I delete four point in a row, Collector crashes.
I would expect it to work in a similar way as it works in Collector Classic. It is much more user friendly.

**Comment by kyliedonia** [comment]
Thanks for the additional info @ananovosel. Which part do you find not intuitive -- how the point is moved by moving the map and then placing it, or that `Update selected point` is in the overflow menu?

**Comment by dmcilwr** [comment]
Did some testing, and my panel/overflow menu issue is dependent on whether the device is in landscape or portrait mode
- in landscape mode no option to resize the panel and no overflow menu
- switch to portrait mode and a resizable panel appears at the bottom of the screen complete with the overflow menu you described. Was able to edit existing vertices as expected 

Note 
- 1st attempt to return to landscape mode crashed the app but has worked okay since
- landscape to portrait, open panel, return to landscape and the panel works only for as long as I stay in that map. If I  return to map selection screen and reopen the first map I have to repeat the tablet flipping process. Also have to repeat for each different map I  open. Also have to repeat every time I close and reopen the app, but not if I minimize and reopen the app while still in the map
-  before the resizable panel is used the first time I'm able to scroll to the bottom of the attribute list AND  all the action options (edit, copy etc) but once the resizable panel has been opened (& minimized) the # of attributes impacts whether I can access all the action options @ 7 attributes, everything is fine, @ 20 attributes and I can only scroll as far as "edit"
